src/action.js

```diff
@@ -165,7 +165,7 @@ async function commit(params) {
     {
       path: path.normalize(questionPath),
       mode: "100644",
-      content: questionData,
+      content: `${questionData}`,
```

Comment:
What do you think of something like:

questionData ?? "Could not fetch question data"

Or the other alternative I was thinking was not adding the question path at all if questionData was null, but I think being more explicit that we couldn't fetch the question data is more clear.

Reply:
Hey Josh, sounds good.
I've updated the change with a fallback message.

Reply:
Awesome, thank you!